src/drive/web/modules/public/LightFileViewer.spec.jsx

```diff
@@ -35,6 +35,8 @@ describe('LightFileViewer', () => {
     })
 
     it('should have the sharing banner and public toolbar but no viewer toolbar', () => {
+      const consoleSpy = jest.spyOn(console, 'error').mockImplementation() // to be removed with https://github.com/cozy/cozy-libs/pull/1457
```

Comment:
on devrait mettre un `TODO: to be removed with...` pour tracker un peu mieux le message, soit via des plugins dans l'IDE, soit via une app github (qu'on a d'installé sur Banks)

Reply:
le plus simple serait de merger l'autre PR (je l'ai approuvé) pour éviter de cas

Reply:
TODO added in #2508

I didn't upgrade cozy-sharing yet. Another PR is needed to go from cozy-sharing@3.12.9 to cozy-sharing@4.1.0